src/garage/sampler/simple_sampler.py

```diff
@@ -0,0 +1,105 @@
+''' Generic sampler can be used with all RL algos #TODO: avnishna update description'''
+from garage.sampler.sampler import Sampler
+import numpy as np
+import time 
+class SimpleSampler(Sampler):
+    """Generic sampler used for off policy sampling
+
+    Args:
+        agent(Policy): Agent used to select actions.
+        env(gym.Env): Environment to perform actions in.
+    Notes:
```

Comment:
its seems like this is redundant with KR's new sampler interface?

Reply:
I think it's fine to leave this for now. I haven't yet refactored everything else around the sampler interface, and when I do I'll refactor this one to be a minimal implementation of the interface (which I want to have anyways).